src/app/components/work-item-list-entry/work-item-list-entry.component.html

```diff
@@ -50,8 +53,8 @@
                 [allowDelete]="false"
                 (onLabelClick)="labelClick($event)"></f8-label>
             </div>
-            <div class="f8-wi__list-description">
-              <span class="color-grey pull-left"
+            <div class="f8-wi__list-description width-100px">
```

Comment:
Why not just add the `width: 100px;` to the `.f8-wi__list-description` class? This is an odd additional class to have here and will require HTML to be updated for what should just be a CSS update.

Reply:
.f8-wi__list-description is generic class for three divs. the last div width is varying from other two and rest of the css is same